src/pages/PointOfContact/index.tsx

```diff
@@ -155,6 +157,12 @@ function PointOfContact(): JSX.Element {
   const navigateSignSubmit = (): void =>
     navigate(`/filing/${year}/${lei}/submit`);
 
+  const handlePhoneExtensionInput = (
+    event: React.ChangeEvent<HTMLInputElement>,
+  ): void => {
+    setValue('phoneExtension', processNumbersOnlyString(event.target.value));
+  };
+
```

Comment:
Is this being unused a part of the functionality that is being left in pending the final decision? If so, you can go ahead and just resolve this.